app/templates/me/index.hbs

```diff
@@ -31,8 +31,8 @@
 
     <p>
         If you want to use package commands from the command line, you will need to
-        login with <code>cargo login</code> using one of the tokens listed below. 
-        For security reasons, you should supply a token when prompted rather than passing it on the command line. 
+        login with <code>cargo login</code> using one of the tokens listed below. <br/>
+        When working in shared environments, it is more secure to supply a token when prompted rather than passing it on the command line. 
```

Comment:
How about mentioning that it's more secure because of being able to access the key via shell history? I'm admittedly not sure how accessing the key via `ps` would work.